source/slang/slang-compiler.cpp

```diff
@@ -1686,159 +1761,160 @@ SlangResult dissassembleDXILUsingDXC(
         case CodeGenTarget::HostCallable:
         case CodeGenTarget::SharedLibrary:
         case CodeGenTarget::Executable:
-            {
-                RefPtr<DownstreamCompileResult> downstreamResult;
+        {
+            RefPtr<DownstreamCompileResult> downstreamResult;
 
-                if (SLANG_SUCCEEDED(emitWithDownstreamForEntryPoint(
-                    compileRequest,
-                    entryPointIndex,
-                    targetReq,
-                    endToEndReq,
-                    downstreamResult)))
-                {
-                    maybeDumpIntermediate(compileRequest, downstreamResult, target);
+            if (SLANG_SUCCEEDED(emitWithDownstreamForEntryPoints(
+                compileRequest,
+                getEntryPointIndices(entryPoints),
+                targetReq,
+                endToEndReq,
+                downstreamResult)))
+            {
+                maybeDumpIntermediate(compileRequest, downstreamResult, target);
```

Comment:
More formatting changes on `switch` statements here that are making it harder to evaluate the actual diffs.